### ISSUE TITLE
Ability to rename files and folders

### ISSUE THREAD

**Issue by stefangordon** [azure-storage/xplat#154]

**Comment by sedouard** [comment]
Strolling through the apis for blob management. I don't see anything to change the name of a blob. Do other tools let you do this / can you do this from other sdks?

**Comment by felixrieseberg** [comment]
Yep, not natively supported. I'm moving this to the 'Next' milestone.

**Comment by stefangordon** [comment]
You just enumerate the folder and delete all the files.  Most clients have the feature.   An alternative that might be easier would be a select all button for the file checkboxes so you could delete the folder yourself.  Might be a hassle with deep folder hierarchies but its better than nothing.

**Comment by stefangordon** [comment]
Fwiw 'rename folder' works the same way (enumerate and rename blobs)

**Comment by felixrieseberg** [comment]
I guess you mean "delete folder", right? Because that's still totally in the 0.4 milestone :santa:

**Comment by felixrieseberg** [comment]
Right, but there's no native rename feature, meaning that people revert back to 'copy' followed by delete. Copying with meta data and ensuring that the copy operation worked is hard, which is why I'm moving this to a next release.

**Comment by stefangordon** [comment]
doh, yes :)  And I agree delete much more important than rename! Thanks!
